packages/jest-changed-files/src/hg.ts

```diff
@@ -20,7 +20,8 @@ const adapter: SCMAdapter = {
 
     const args = ['status', '-amnu'];
     if (options && options.withAncestor) {
-      args.push('--rev', `min((!public() & ::.)+.)^`);
+      // args.push('--rev', 'min((not public() and ancestors(.))^ or .^)');
+      args.push('--rev', `min((!public() & ::.)+.)^1`);
```

Comment:
`min((!public() & ::.)+.)^` and `min((!public() & ::.)+.)^1` are equivalent. From the docs: `x^ – Equivalent to x^1, the first parent of each changeset in x.` Without `1` it does not work nor locally, neither in CI (throws: `abort: empty revision range`). Works with `1`, but test fails!